### PR TITLE
[CBRD-22784] Large json array index causes unclear invalid path error

### PR DESCRIPTION
**Pull request by radudoros**
http://jira.cubrid.org/browse/CBRD-22784

Added new Invalid json path; index too large error (-1207).

### CODE REVIEW

**Comment by eseokoh** [comment]
What about separating an error code and message for *larger_than_max_array_idx_msg* for localization?

File: src/query/arithmetic.c [hunk 1]
```diff
@@ -6278,11 +6278,15 @@ db_evaluate_json_search (DB_VALUE *result, DB_VALUE * const * args, const int nu
 
   JSON_DOC *result_json = nullptr;
   if (paths.size () == 1)
-    {      
-      size_t escaped_size;
+    {
+      error_code = db_json_path_unquote_object_keys_external (paths[0]);
```
Comment (ribram) [inline src/query/arithmetic.c]: fix indentation

File: src/query/arithmetic.c [hunk 2]
```diff
@@ -6303,17 +6307,22 @@ db_evaluate_json_search (DB_VALUE *result, DB_VALUE * const * args, const int nu
   result_json_owner.create_mutable_reference ();
   for (std::size_t i = 0; i < paths.size (); ++i)
     {
-      JSON_DOC *json_array_elem = nullptr;
+      error_code = db_json_path_unquote_object_keys_external (paths[i]);
```
Comment (ribram) [inline src/query/arithmetic.c]: fix indentation

**Comment by eseokoh** [comment]
@radudoros 

The expected answer for the failure case should be updated.
I think you can merge the fix :)

**Pull request merged**